pages/rooms/index.tsx

```diff
@@ -57,6 +57,14 @@ export default function Rooms({
     clientToken: '',
     refreshToken: '',
   });
+  const [optionalFeatures, setOptionalFeatures] = useState<{
```

Comment:
Do we need another state here? When the prop changes it will rerender, right?

Reply:
Thanks for pointing it out. It is useless. 🙂

Resolved in [bb1ef94](https://github.com/team-telnyx/telnyx-meet/pull/96/commits/bb1ef94e86167cbbe3f2fa45608e07990002b776)

Reply:
Thanks for pointing it out. It is useless having this state here. 🙂

Resolved in [bb1ef94](https://github.com/team-telnyx/telnyx-meet/pull/96/commits/bb1ef94e86167cbbe3f2fa45608e07990002b776)